README.md

```diff
@@ -0,0 +1,199 @@
+comeurop
+========
+
+Pour déployer rapidement un projet, voir : [Déploiement rapide](#deploiement_rapide)
```

Comment:
Put the "rapid deployment" section right at the beginning of the document (it's a common practice).